.stylelintrc

```diff
@@ -14,7 +14,8 @@
     "selector-type-no-unknown": [
       true,
       {
-        "ignore": ["custom-elements"]
+        "ignore": ["custom-elements"],
+        "ignoreTypes": ["overlay", "container"]
```

Comment:
Does stylelint flag those for `ReactModal`? Sounds like a false positive there

Reply:
Yup it does